src/manual/design/namespaces.md

```diff
@@ -103,15 +106,12 @@ Both are listed here:
    Roles of this form represent the scopes available to members of the given Mozilla LDAP group via the login service.
 
 * `mozilla-user:<userName>` -
```

Comment:
Isn't this done via `login-identity:<lI>` roles now?

Reply:
Could you point me where this done? In `tc-login` we implement the `mozilla-user:<username>` role in https://github.com/taskcluster/taskcluster-login/commit/ea7ac8ff862b9540f2473fa2ffb97d1dc8989f94#diff-43d1529de0fc08f33b7f5755b6916dc8R238.

Reply:
Oh, that's right!